packages/lib-classifier/src/store/WorkflowStore/Workflow/Workflow.js

```diff
@@ -81,11 +81,13 @@ const Workflow = types
       return anyTranscriptionTasks
     },
 
-    get usesFreehandLineTool() {
+    get usesMachineLearnt() {
+      const toolTypes = ['circle', 'ellipse', 'freehandLine', 'line', 'point', 'polygon', 'rectangle', 'rotateRectangle'];
```

Comment:
This whitelist kind of breaks the idea that `plugins/drawingTools` should be a self-contained package, and the list itself needs to be maintained by hand whenever new drawing tools are added. 

Maybe delegate responsibility to individual `Tool` objects? Eg. make each tool responsible for its own Caesar reducer. 

```js
const taskUsesMachineLearning = task.tools.some(tool => tool.caesarReducer === 'machineLearnt')
```

Reply:
I see what you're saying.. I like that approach, yet I'd want to have a broader team discussion about more robust Caesar integration. Especially considering Shaun's comments above about knowing if/when to make a Caesar request.

Reply:
True. You might also be able to reflect on the task type with `getType(task)`, since `TranscriptionTask` and `DrawingTask` are different types, with different reducers. I think that, right now, the task models only allow one reducer per task.

Reply:
So something like either a Facade or a Strategy pattern, where you have a generic Caesar client and tools or tasks generate specific strategies for generating client queries.